src/main/java/com/virtuslab/using_directives/custom/Parser.java

```diff
@@ -185,7 +185,10 @@ String key() {
 
   SettingDefOrUsingValue valueOrSetting() {
     if (literalTokens.contains(in.td.token)
-        || (in.td.token == Tokens.IDENTIFIER && in.td.name.equals("-"))) {
+        || (in.td.token == Tokens.IDENTIFIER && in.td.name.equals("-"))
```

Comment:
Can you add a test case for this?

Reply:
What should it test?

Reply:
I assumed that it's needed to fix some specific case, right?

Reply:
Actually, the default true value is a very very specific case and we need here to make sure that we cannot accept the next token. (The next token doesn't fit to any path of parsing)

Reply:
ok, nvm then